webrender/src/device.rs

```diff
@@ -1269,7 +1253,7 @@ impl Device {
     pub fn free_texture_storage(&mut self, texture: &mut Texture) {
         debug_assert!(self.inside_frame);
 
-        if texture.format == ImageFormat::Invalid {
+        if texture.width + texture.height == 0 {
```

Comment:
This made me do a double take, but I guess it works :)

Reply:
As far as I see, the only case it tries to handle is when the texture wasn't ever given contents (leaving the size to 0 and the format to `Invalid`).